egg/core/callbacks.py

```diff
@@ -131,11 +134,23 @@ def __init__(
         checkpoint_path: Union[str, pathlib.Path],
         checkpoint_freq: int = 1,
         prefix: str = "",
+        start_epoch: int = 0,
+        max_chk: int = sys.maxsize,
     ):
+        """Saves a checkpoint file for training
+        Arguments:
+            checkpoint_path {Union[str, pathlib.Path]} -- path to checkpoint directory, will be created if not present
```

Comment:
Please use [this](https://github.com/facebookresearch/EGG/blob/master/egg/core/reinforce_wrappers.py#L621-L629) style (with a full stop at the end of method description).

Reply:
fixed in 6e029ca